index.html

```diff
@@ -187,7 +187,7 @@ <h3 id="community" data-export="" data-dfn-type="dfn">
 they can also be used for spreading misinformation,
 revealing private personal information (doxing),
 harassment, and persecution.
-We will consider these risks in the work we do,
+We will mitigate these risks in the work we do,
```

Comment:
Again, I'm not sure how you can make this statement true.  I don't think it's possible absolutely prevent doxing, for example.